tools/packrom/packrom.cpp

```diff
@@ -93,7 +100,7 @@ u32 crc_tab[256];
  */
 u32 chksum_crc32 (unsigned char *block, unsigned int length)
 {
-   unsigned long crc;
+   register unsigned long crc;
```

Comment:
The register keyword is ignored and should not be used because it is deprecated and causes build warnings.

```
g++ -c packrom.cpp -o Debug/packrom.o -DLINUX -Wall -g -MD -MP -MF Debug/packrom.d -DDEBUG
packrom.cpp: In function ‘u32 chksum_crc32(unsigned char*, unsigned int)’:
packrom.cpp:103:27: warning: ISO C++17 does not allow ‘register’ storage class specifier [-Wregister]
  103 |    register unsigned long crc;
      |                           ^~~
```